bench/run-per-pubkey-sweep.sh

```
@@ -0,0 +1,126 @@
+# Clients default to 2-3 relays per author based on fiatjaf-only 7d Greedy data.
+# This sweep validates the sweet spot at 1yr with FD+Thompson and plain FD.
+#
+# Phase 1: Reduced sweep (3 profiles × 3 rpu × 5 sessions = 45 runs)
+#   to detect the trend before committing to the full matrix.
+# Phase 2 (optional): Full sweep (6 profiles × 5 rpu × 5 sessions = 150 runs per algo)
+#
+# Note: some authors have fewer than 5 write relays. The script reports
+# the distribution of declared write relay counts for each profile.
+set -uo pipefail
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)